### PR TITLE
rebase on master

### PR DESCRIPTION
**Pull request by gamerson**

### CODE REVIEW

File: tools/plugins/com.liferay.ide.upgrade.plan.core/src/com/liferay/ide/upgrade/plan/core/UpgradeTaskStep.java [hunk 1]
```diff
@@ -32,8 +38,23 @@
...
 
+	public default UpgradeTaskStepStatus getStatus() {
+		return UpgradeTaskStepStatus.INCOMPLETE;
+	}
+
 	public String getTaskId();
 
 	public String getUrl();
 
+	public default IStatus perform(IProgressMonitor progressMonitor) {
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.upgrade.plan.core/src/com/liferay/ide/upgrade/plan/core/UpgradeTaskStep.java:49]: We don't need a perform for a task step.

**Comment by christopherbryanboyd** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by christopherbryanboyd** [comment]
Pull request submitted to https://github.com/gamerson/liferay-ide/pull/4113. [See changes here.](https://github.com/christopherbryanboyd/liferay-ide/compare/47fa31cc2~1...47fa31cc2) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by christopherbryanboyd** [comment]
Pull request submitted to https://github.com/gamerson/liferay-ide/pull/4114. [See changes here.](https://github.com/christopherbryanboyd/liferay-ide/compare/ac24d883f~1...ac24d883f) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**